ConwaysGameOfLife/Universe.cs

```diff
@@ -74,7 +74,7 @@ public int CountLiveNeighbours(Cell cell)
             var neighbourLocations = GetCellNeighbourLocations(cell);
             foreach (var neighbour in neighbourLocations)
             {
-                var state = GetCellAtLocation(neighbour.X, neighbour.Y).CellState;
+                var state = GetCellAtLocation(neighbour).CellState;

```

Comment:
```suggestion
                var cell = GetCellAtLocation(neighbour);
                var state = cell.CellState;
```

as you had done above